pkg/subctl/cmd/deploybroker.go

```diff
@@ -117,7 +117,7 @@ var deployBroker = &cobra.Command{
 		}
 
 		err = subctlData.WriteToFile(brokerDetailsFilename)
-		status.End(err == nil)
+		status.End(0)
```

Comment:
not equivalent

Reply:
err == nil is true which is success bool. and here also 0 is success.

Reply:
`err == nil` is `true` only if `err` is `nil`; you’d need something like `err == nil ? success : failure` (but as a function, since Go doesn’t have `? :`).